README.md

```diff
@@ -2,4 +2,114 @@
 
 ## Marine Rust SDK
 
-This SDK is intended to run backend application on the Fluence network with [Marine](https://github.com/fluencelabs/marine) runtime. More information about usage and some internals could found in [docs](https://fluence.dev/docs/marine-book/marine-rust-sdk/).
+This SDK empowers developers to create suitable general-purpose Wasm services executable by the [Marine](https://github.com/fluencelabs/marine) runtime. Such services are constructed from one or more Wasm modules with the help of [interface-types](https://github.com/WebAssembly/interface-types) and a [shared-nothing linking](https://training.linuxfoundation.org/blog/how-and-why-to-link-webassembly-modules/) scheme. Each module results from Rust code compiled to `wasm32-wasi` compile target with some glue code auto-generated by this SDK. The primary purpose of the SDK influenced its design is to provide developers a way to write Wasm modules on "vanilla" Rust, wrapping necessary blocks with the supplied macros.
```

Comment:
how about:
This SDK empowers developers to create general-purpose Wasm IT modules executable by the [Marine](https://github.com/fluencelabs/marine) runtime from "vanilla" Rust. The SDK provides necessary Rust macros and glue-code to compile Rust code to `wasm32-wasi` modules. The resulting Wasm modules can be combined into services with the help of [interface-types](https://github.com/WebAssembly/interface-types) and a [shared-nothing linking](https://training.linuxfoundation.org/blog/how-and-why-to-link-webassembly-modules/) scheme.

Reply:
In this variant, it's not clear what a service is, how about this one


This SDK empowers developers to create general-purpose Wasm modules and combine them in a multi-module setup with the help of interface-types and shared-nothing linking scheme. The SDK provides all necessary macros and other features to make Wasm developing process as close as possible to the one with the "vanilla" Rust. Compiled modules are intended to run with the Marine runtime.

Here I tried to omit services and use Wasm/Rust neutral words.